frontend/src/styles/home/FeaturedProjectCard.module.css

```diff
@@ -85,19 +68,72 @@
   transition: color 300ms ease;
 }
 
+@media screen and (min-width: 700px){
 
-@media screen and (max-width: 1000px){
   #cardContainer {
-    display: block;
-    margin-left: 3em;
-    margin-right: 3em;
+    display: flex;
+    flex-direction: row;
+    margin-bottom: 5vw;
+  }
+
+  #cardImageContainer {
+    display: flex;
+    flex: 0.35;
+    margin-bottom: 0;
+  }  
+
+  #cardTextContainer {
+    flex: 0.65;
+    right: 0;
+    top: 0;
+    margin-left: 2vw;
+    margin-top: 0;
+    flex-direction: column;
+    height: 200px;
+  }
+
+  #cardVariableTextContainer {

```

Comment:
im guessing you consulted katherine about this change?

Reply:
i.e that it *is* intended behaviour for "see more" link to clamp to bottom of card?

Reply:
@heylookitsme This change was done to fix some styling issues. With this implementation, the text above the show more button (the project title + descriptions) can't overflow past the level of the picture. This was causing some ugly styling issues at certain screen sizes before and also was a reason for overlapping text.